projects/batfish/.project

```diff
@@ -7,16 +7,6 @@
 	<buildSpec>
 		<buildCommand>
 			<name>org.eclipse.ui.externaltools.ExternalToolBuilder</name>
```

Comment:
OK I figured it out. I sometimes disable the xtext builder because it causes things to slow down. Disabling that builder causes two effects:
1. It creates a file: projects/batfish/.externalToolBuilders/org.eclipse.xtext.ui.shared.xtextBuilder.launch
2. It modifies projects/batfish/.project
To avoid propagating my disabling of xtext builder to the repo, i put the file in 1. in .gitignore. However, I didn't realize that .project in 2. was also being modified. The fix was to tell git to ignore changes to .project when I feel like disabling the xtext builder. This is accomplished with:
git update-index --assume-unchanged projects/batfish/.project
before making the change.